dags/imagegen_devx/configs/gke_config.py

```diff
@@ -25,24 +25,28 @@
     "v4-8": TpuVersion.V4,
     "v4-16": TpuVersion.V4,
     "v5-8": TpuVersion.V5P,
+    "v6e-256": TpuVersion.TRILLIUM,
```

Comment:
Sorry I may miss this file a while back, but why not re-use what we have in `vm_resource.py`? We share all GCE and GKE resources in `vm_resource` file.

Reply:
@RissyRan We are using the resources defined in vm_resource. This file simply maintains the mapping, which helps resolve clusters, device types, etc., in test cases.

Reply:
Cool, this also makes as it's under imagegen_devx folder, not in general.